networkx/algorithms/disjoint_paths/simple_disjoint.py

```diff
@@ -0,0 +1,73 @@
+"""Disjoint paths computation using removal of found shortest paths.
+"""
+
+#    Copyright (C) 2015-2016 by
+#    Niels L. M. van Adrichem <n.l.m.vanadrichem@tudelft.nl>
+#    All rights reserved.
+#    BSD license.
+
+__author__ = """\n""".join(['Niels L. M. van Adrichem <n.l.m.vanadrichem@tudelft.nl>'])
+__all__ = ['simple_disjoint']
+
```

Comment:
The head is not precisely as it should be according to networkx standards. Please see https://github.com/networkx/networkx/wiki#user-content-top-of-the-module